test/passing/extensions-indent.ml.ref

```diff
@@ -209,3 +209,87 @@ let () =
        (* 2 *)
        [@foo "bar"]
        (* 3 *)]
+
+open%ext M
+
+open%ext M
+
+[%%ext open! M]
+
+include%ext M
+
+include%ext M
+
+let x =
+  let open%ext M in
+  x
+
+let x =
+  [%ext
+       let open M in
+       x]
+
+let x =
+  [%ext
+       let open! M in
+       x]
+
+exception%ext E
+
+exception%ext E
+
+let _ =
+  let exception%ext E in
+  x
+
+let _ =
+  [%ext
+       let exception E in
+       x]
+
+module%ext E = P
+
+module%ext E = P
+
+module%ext rec K = A
+and L = A
+
+module%ext rec K = A
+and L = A
+
+let _ =
+  let module%ext E = P in
+  x
+
+let _ =
+  [%ext
+       let module E = P in
+       x]
+
+module type%ext E = P
+
+module type%ext E = P
+
+class%ext x = y
+
+class%ext x = y
+
+class%ext x = y
+
+   and y = z
```

Comment:
Why is this indented ?

Reply:
it was an issue with the existing boxes, it's fixed